packages/core/npm-aqua-compiler/src/imports.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Arborist from "@npmcli/arborist";
+import { breadth } from "treeverse";
+
+export async function gatherImportsFromNpm(
+  path: string,
+): Promise<Record<string, string[]>> {
+  const arb = new Arborist({ path });
+
+  return await gatherImportsFromArborist(arb);
+}
+
+export async function gatherImportsFromArborist(
```

Comment:
what is the reason to have this as a separate function? do we plan on using it separately from the `gatherImportsFromNpm` function?

Reply:
Agree. Probably we need a latter function only